xml/dm_registers.xml

```diff
@@ -592,7 +592,7 @@
         If resume groups are not implemented, then \FdmDmcsTwoGrouptype will
         remain 0 even after 1 is written there.
 
-        The external triggers available to add to halt groups may or may not
+        The external triggers available to add to halt groups may
         be distinct from the external triggers available to add to resume groups.
```

Comment:
How about "... halt groups may be the same as or distinct from the external triggers..."

Reply:
OK.  Done.